ngrok-java/src/main/java/com/ngrok/Session.java

```diff
@@ -33,47 +57,131 @@ public static Session connect(Builder builder) throws IOException {
         }
     }
 
+    /**
+     * Returns the metadata for the session.
+     *
+     * @return the metadata for the session
+     */
     public String getMetadata();
 
+    /**
+     * Returns a new {@link TcpTunnel} instance with the default settings.
+     *
+     * Returns a new {@link TcpTunnel} instance with the default settings
+     * 
+     * @throws IOException if an I/O error occurs during the tunnel creation
+     */
     public default TcpTunnel tcpTunnel() throws IOException {
         return tcpTunnel(new TcpTunnel.Builder());
     }
 
+    /**
+     * Returns a new {@link TcpTunnel} instance with the specified settings.
```

Comment:
We should expand on these - it doesn't just create a new instance, it actually creates a new tunnel and returns it.